Plugins/Persistence/Retrieve-Startup.ps1

```diff
@@ -183,21 +141,22 @@ process {
     # Iterate through each user profile grabbing the artifacts
     foreach ($User in $Users){
 
-        foreach ($Artifact in $UserArtifacts) {
+        foreach ($UserArtifact in $UserArtifacts) {
 
-            $ScriptBlock = $ExecutionContext.InvokeCommand.NewScriptBlock(("& 'C:\ProgramData\{0}' fs --accessor ntfs cp '\\.\{1}\{2}' C:\ProgramData\{3}") -f ((Split-Path $Velo_exe -Leaf), $User, $Artifact, $Session.ComputerName))
-            Invoke-Command -Session $Session -ScriptBlock $ScriptBlock -ErrorAction SilentlyContinue | Out-Null
+            $Artifact = ('{0}\{1}' -f $User,$UserArtifact)
+
+            Copy-PRItem -Session $Session -Path $Artifact -Destination ("C:\ProgramData\{0}" -f $Session.ComputerName)
```

Comment:
Also looks like we are using `C:\ProgramData\COMPURTERNAME` here where we used `C:\ProgramData\Power-Response` on line 129 above.